src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.NET.Sdk.targets

```diff
@@ -132,7 +132,6 @@ Copyright (c) .NET Foundation. All rights reserved.
                                        RuntimeConfigDevPath="$(ProjectRuntimeConfigDevFilePath)"
                                        RuntimeIdentifier="$(RuntimeIdentifier)"
                                        PlatformLibraryName="$(MicrosoftNETPlatformLibrary)"
-                                       RuntimeFrameworkVersion="$(RuntimeFrameworkVersion)"
```

Comment:
Can we remove the property from the C# Task as well?  That way it isn't dead code that someone in the future has to figure out where it came from and is it still used.